src/main/pg/autopilot_multirotor.h

```diff
@@ -28,20 +28,20 @@
 #include "pg/pg.h"
 
 typedef struct apConfig_s {
-    uint8_t landing_altitude_m;   // altitude below which landing behaviours can change, metres
-    uint16_t hover_throttle;      // value used at the start of a rescue or position hold
-    uint16_t throttle_min;
-    uint16_t throttle_max;
-    uint8_t altitude_P;
-    uint8_t altitude_I;
-    uint8_t altitude_D;
-    uint8_t altitude_F;
-    uint8_t position_P;
-    uint8_t position_I;
-    uint8_t position_D;
-    uint8_t position_A;
-    uint8_t position_cutoff;
-    uint8_t max_angle;
+    uint8_t ap_landing_altitude_m;   // altitude below which landing behaviours can change, metres
+    uint16_t ap_hover_throttle;      // value used at the start of a rescue or position hold
+    uint16_t ap_throttle_min;
+    uint16_t ap_throttle_max;
```

Comment:
```suggestion
    uint16_t ap_hover_throttle;      // value used at the start of a rescue or position hold
    uint16_t ap_throttle_min;
    uint16_t ap_throttle_max;
    uint8_t ap_landing_altitude_m;   // altitude below which landing behaviours can change, metres
```
should save several bytes